keps/sig-architecture/1618-conformance-profiles/README.md

```diff
@@ -0,0 +1,444 @@
+packaged as a Helm chart may use a PersistentVolume. Because that feature is not
+part of the conformance program, there are valid, Certified Kubernetes
+distributions that do not support it, and therefore the application deployment
+will not succeed.
+
+Even as we work to expand those behaviors to include all non-optional, GA
+features, there is a substantial amount of functionality that may not apply to
+all cluster deployments. For example, edge use cases may intentionally not offer
+any stateful services. It would be beneficial for those deploying applications
+to easily identify sets of functionality that are needed for their application,
+and confirm that a given vendor offering (if not a specific cluster) is able to
+support the application.
+
+In this context, “optional” may be from a given user's point of view. For
+example, the smallest, “base” profile should provide functionality that at a
+minimum is capable of running stateless user workloads. From this user’s point
+of view, cluster administration functions are not required.
+
+Profiles also support a simplified model for determining if a given application
+that has been packaged to run on Kubernetes (e.g., in a Helm chart) is able to
+run on a given cluster. If clusters are classified into a small (~5-10) number
+of profiles that define broad functional boundaries, applications will be able
+to document the profiles required in order for them to run.
+
+### Goals
+
+ * Design and implement a mechanism to express profiles in the conformance
+   behavior definitions.
+ * Design and implement a mechanism to validate whether a cluster conforms to
+   each profile.
+ * Document guidance on how to choose the scope of individual profiles.
+ * Document guidance on how to decide the profile in which a new feature
+   belongs.
+ * Define a base profile and at least one other profile.
+
+### Non-Goals
+
+ * Define a complete set of profiles that covers all behaviors. This is an
+   ongoing effort for the conformance subproject.
+ * Complete all tests to validate all profiles. This is an ongoing effort for
+   the conformance subproject.
+
+## Proposal
+
+<!--
+This is where we get down to the specifics of what the proposal actually is.
+This should have enough detail that reviewers can understand exactly what
+you're proposing, but should not include things like API designs or
+implementation.  The "Design Details" section below is for the real
+nitty-gritty.
+-->
+
+### User Stories (optional)
+
+<!--
+Detail the things that people will be able to do if this KEP is implemented.
+Include as much detail as possible so that people can understand the "how" of
+the system.  The goal here is to make this feel real for users without getting
+bogged down.
+-->
+
+#### Role: Developer
+
+As a developer of an optional feature, I must be able to define behaviors for
+that feature that a vendor claiming support for the feature must follow.
+
```

Comment:
Whose responsibility is it to assign optional features to Profiles / create new Profiles?  

I feel like as a developer I need to know this information if I want my feature to be included in Conformance testing.  

OTOH, maybe inclusion in a Profile is up to someone else to decide